src/modules/auth/auth.controller.ts

```diff
@@ -13,5 +14,64 @@ export class AuthController {
     return this.authService.register(body);
   }
 
+  @Post('login')
+  async login(@Body() body: LoginDto, @Res() res: ExpressResponse) {
+    try {
+      const { accessToken, refreshToken } = await this.authService.login(body);
+      this.setCookies(res, accessToken, refreshToken);
+      res.status(HttpStatus.OK).end('đăng nhập thành công');
```

Comment:
Dont need messages for these apis. The statuses already have their meaning.

Besides, do not use Vietnamese in system messages.